src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import {
+  Grid,
+  Avatar as AvatarMaterialUI,
+  makeStyles,
+  CircularProgress,
+} from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import { AvatarProps } from './types';
+
+const useRegularTypeStyles = makeStyles({
+  avatar: {
+    margin: 10,
+    width: 60,
```

Comment:
@michaelmoney here you are using `px`. Wouldn't be better to convert it to `em`? This applies to all styles :)

Reply:
I've change `px` to `rem` values here: https://github.com/BuildIt-Poland/buddy-app/pull/18/commits/81f2a972b03b57fabeb5645aa03da2fdbc068ab2